assets/components/account-menu/_account-menu.scss

```diff
@@ -19,17 +19,16 @@
 }
 
 .service-info {
-  border-top: none;
   display: block;
   position: relative;
   z-index: 997;
+  overflow: hidden;
```

Comment:
Is this necessary? It doesn't look like it was ever needed before.

Reply:
@rpowis yes it is necessary with how the 10px top border was previously implemented. The reason it was changed from being applied on the account-menu and hidden on the service-info container was because it was causing an issue with the sub-nav reveal. As the element is contained in the service-info it was displaying over the top of it so hid the blue border. That implementation meant the blue border now didn't show on pages without the account-menu. Applying overflow: hidden now means that the sub-nav is contained to the service-info padding-box so the blue border can be applied on the service-info element again.

Reply:
Gotcha. I didn't realise it was to counter something in account menu. Thanks for clarifying 👍